packages/backend/src/accounting/utils.ts

```diff
@@ -1,10 +1,9 @@
-import { AssetAccount, Balance } from './service'
+import { AssetAccount } from './service'
 
-const ASSET_ACCOUNTS_RESERVED = 32
+const ASSET_ACCOUNTS_RESERVED = 8
```

Comment:
What is this constant for?

Reply:
It's used in generating deterministic asset account ids:
https://github.com/interledger/rafiki/blob/main/packages/backend/src/accounting/utils.ts#L28-L30
We're now down two types of asset accounts (liquidity & settlement).

I was actually just considering replacing it with:
```
const MAX_ASSETS = 2**16
```
based on Tigerbeetle's two byte unit field. And then `getAssetAccountId` could do:
```
BigInt(type * MAX_ASSETS + unit)

Reply:
Couldn't miscellaneous account uuids get generated into that range?

Reply:
Yeah, which is why I was still favoring `ASSET_ACCOUNTS_RESERVED` over `MAX_ASSETS` since the range is limited by the number of assets.

However, I _think_ we would safely fail to create an asset if it's id is already used.
https://github.com/interledger/rafiki/blob/main/packages/backend/src/asset/service.ts#L57-L68
We'd end up with a unit sequence gap, and retrying would attempt to create the asset with the next unit value.

Reply:
I also had the thought that `validateId` in `accountingService.createAccount` could disallow ids in the possible asset id range.